R/build.r

```diff
@@ -108,8 +108,12 @@ readme <- function(pkg = ".") {
 
 copy_bootstrap <- function(pkg = ".") {
   pkg <- as.sd_package(pkg)
-
-  bootstrap <- file.path(inst_path(), "bootstrap")
+  user_bootstrap <- pkg$bootstrap_path
+  if(file.exists(user_bootstrap)){
```

Comment:
Can you please put spaces around `()` and `{}`? (See http://r-pkgs.had.co.nz/r.html#style)